contract_journal/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+========================
+Contract Account Journal
+========================
+
+This module was written to extend the functionality of contracts to support to
+set an account journal on contract.
```

Comment:
This module allows to set a sales account journal for creating the invoices on it.